.flake8

```diff
@@ -12,7 +12,8 @@ ignore = E203, E501, W503,  # handled by black
          S311,
          T001,
          B007
-         S608
+         S608,
+         S105
```

Comment:
What is this warning about?

Reply:
I suppose in settings.py we had SECRET_KEY and flake8 does not like we store such secure data in open source project.
S105 Possible hardcoded password.

Reply:
Secret key get access to use my app for other people. I found good practise is to store secret key in terms of environment variable 
 `SECRET_KEY = os.environ.get("SECRET_KEY")`
 Fixed in my project.

Reply:
> Fixed in my project.
 
You can remove this line now.